src/config.ts

```diff
@@ -64,11 +64,17 @@ export interface Config extends AuthenticationConfig {
 
 // Default values for configuration for a profiler.
 export const defaultConfig: Config = {
-  projectId: '{{projectId}}',
   logLevel: 1,
   serviceContext: {
     version: '',
   },
   disableHeap: false,
   disableTime: false,
+  instance: '',
+  zone: ''
+};
+
+// Configuration parameters set internally, not by users.
+export const internalConfig = {
```

Comment:
You don't need this here. Simply add it to the Service object config here: https://github.com/nolanmar511/cloud-profiler-nodejs/blob/41428d20b54112e67c39888ceef56f5732f89506/src/profiler.ts#L46